openmc/data/library.py

```diff
@@ -23,13 +23,16 @@ class DataLibrary(EqualityMixin):
     def __init__(self):
         self.libraries = []
 
-    def get_by_material(self, name):
+    def get_by_material(self, name, data_type='neutron'):
         """Return the library dictionary containing a given material.
 
         Parameters
         ----------
         name : str
             Name of material, e.g. 'Am241'
+        data_type : str
+            Name of cross section type, e.g. 'neutron', 'photon', 'wmp',
+            'thermal', etc
```

Comment:
There are no others currently so 'etc' doesn't really make sense
```suggestion
            or 'thermal'
```